ocp4-logging/.openshift/templates/README.md

```diff
@@ -0,0 +1,32 @@
+# ClusterLogging Template
+
+Below you can see the parameters that are available for the `ClusterLogging` template. This will show their name, a description and the defaults if applicable
+
+## Parameters
+
+| Name      | Description   | Default |
+| ----------- | ----------- | ------- |
+| ELASTICSEARCH_REDUNDANCY_POLICY | Sharding policy to be used with Elasticsearch (FullRedundancy, MultipleRedundancy, SingleRedundancy, ZeroRedundancy) | SingleRedundancy
+| ELASTICSEARCH_NODE_COUNT | Number of Elasticsearch node to deploy        | 2 |
```

Comment:
Sorry, still additional spaces in the second field

Reply:
Fixed.